build-tools/scripts/y2metainfo

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+#
+# y2metainfo - Automagically generate appstream data for YaST modules based on their desktop files
```

Comment:
Magic is bad for long term maintenance. Can we just drop the word from this comment? (I don't see actual magic in here)

Also let's mention https://www.freedesktop.org/wiki/Distributions/AppStream/ in a comment

Reply:
And also https://features.opensuse.org/319035